services/workflows-service/scripts/workflows/dynamic-ui-workflow.ts

```diff
@@ -104,6 +105,17 @@ export const dynamicUiWorkflowDefinition = {
             },
             { target: 'pending_kyc_response_to_finish' },
```

Comment:
what happens here where its the second kyc is submitted, will the rule catch it?